docs/shared/conf.py

```diff
@@ -63,8 +64,8 @@ def add_base(paths):
 master_doc = 'index'
 
 # General information about the project.
-project = u'edX'
-copyright = u'2013, EdX Doc Team'
+project = 'edX'
+copyright = '{year}, EdX Doc Team'.format(year=datetime.datetime.now().year)
```

Comment:
Is there a script like run_tests.sh from edx_documentation, that I could use here to generate the build directory, or should I start learning sphinx from scratch?

I'd like to see which conf.py files affect the output and change them accordingly.